source/installation.txt

```diff
@@ -90,25 +90,33 @@ command:
 Install from Source
 ~~~~~~~~~~~~~~~~~~~
 
-If the above options still do not allow you to install ``pymongoarrow`` on your
+If you can't use the above options to install ``pymongoarrow`` on your
 system, you can install from source. To learn how, see the `Contributing Guide
 <https://github.com/mongodb-labs/mongo-arrow/blob/main/bindings/python/CONTRIBUTING.md#installing-from-source>`__.
 
 Dependencies
 ^^^^^^^^^^^^
 
-{+driver-short+} requires the following:
+{+driver-short+} requires the following dependencies:
 
-- PyMongo>=4.4
-- PyArrow>=16,<16.1
+- PyMongo version 4.4 or later
+- PyArrow version 16
+
+Installing from source on Linux requires the following additional dependencies: 
+
+- GCC version 12 or later
+- CMake
+- pkg-config
 
 To use {+driver-short+} with a PyMongo feature that requires an optional
-dependency, users must install PyMongo with the dependency manually.
+dependency, you must install PyMongo with the dependency manually.
```

Comment:
I: I'm not certain what this means. Install PyMongo and manually include the needed dependency?

Reply:
I think it means that they need to manually set the dependency option while installing pymongoarrow. The code sample included indicates that it's an option so I'm going to use the option wording here--let me know if that's still confusing.

Reply:
It's actually a little confusing because the docs we point them to [about dependancies](https://pymongo.readthedocs.io/en/stable/installation.html) phrase it as an "automatic installation" but I think I'm leaning towards still calling it manual as they have to pass in a specific command and it doesn't just install with pymongo.